controllers/apply-next/views/step.njk

```diff
@@ -46,8 +46,7 @@
                 {% endif %}
 
                 <div class="form-actions">
-                    <a href="{{ formBaseUrl }}/summary" class="btn btn--outline">Skip to summary</a>
```

Comment:
Can you leave this in? Been helping me debug things and I suspect we'll want a link to it to be present on all pages at some point.